### ISSUE TITLE
Unable to set `parse_mode` for Telegram integration to HTML or plain text

### ISSUE THREAD

**Issue by plyul** [VictoriaMetrics/operator#506]
Chart: victoria-metrics-operator-0.11.3
AppVersion: 0.26.3

I've created Alertmanager (0.24.0) instance via `VMAlertmanager` resource, it's fine.

Then trying to apply `VMAlertmanagerConfig`:
```
  apiVersion: operator.victoriametrics.com/v1beta1
  kind: VMAlertmanagerConfig
  metadata:
    name: test
  spec:
    route:
      receiver: test_channel
      group_by: ['alertname']
      group_interval: 1m
      group_wait: 5s
      repeat_interval: 1m
    receivers:
      - name: test_channel
        telegram_configs:
          - api_url: https://api.telegram.org
            bot_token:
              name: "alertmanager-telegram-bot-token"
              key: botkey
            send_resolved: true
            chat_id: -XXXXXXXXXXXXX
```

The above manifest leads to generating Alertmanager's config with key `receivers[].telegram_configs[].parse_mode` equaling `MarkdownV2`.

If `parse_mode: ""` or `parse_mode: "HTML"` are explicitly added to VMAlertmanagerConfig, then applying fails with error `Unsupported value`.

It seems there is a typo in VMAlertmanagerConfig CRD:

```
parse_mode:
  Description:  ParseMode for telegram message, supported values are MarkdownV2, Markdown, Markdown and empty string for plain text.
  Enum:
    MarkdownV2
    Markdown
    Markdown
  Type:  string
```

**Comment by denisgolius** [comment]
Hi @plyul !
It will be fixed soon :)

**Comment by f41gh7** [comment]
Was fixed at v0.27 release, feel free to re-open if issue wasn't solved.
